src/scripts/components/icons/twitter-icon.tsx

```diff
@@ -2,10 +2,14 @@ import React, {FunctionComponent} from 'react';
 
 export const TwitterIcon: FunctionComponent = () => (
   <svg
-    width="24"
-    height="24"
-    viewBox="0 0 24 20"
+    width="27"
```

Comment:
Could you please check the icons again, something seems a bit off here. I guess it's because the copy icon is not vertically centered. Also a bit more space between the buttons would look better imo.
<img width="711" alt="Screenshot 2020-07-29 at 08 35 35" src="https://user-images.githubusercontent.com/1611635/88765366-ac94cb00-d176-11ea-8229-8a64588b1317.png">